spark/src/main/scala/com/alibaba/graphar/reader/EdgeReader.scala

```diff
@@ -38,6 +39,43 @@ class EdgeReader(prefix: String,  edgeInfo: EdgeInfo, adjListType: AdjListType.V
     throw new IllegalArgumentException
   }
 
+  /** Load the total number of src/dst vertices for this edge type. */
+  def readVerticesNumber(): Long = {
+    val file_path = prefix + "/" + edgeInfo.getVerticesNumFilePath(adjListType)
+    val number = FileSystem.readValue(file_path, spark.sparkContext.hadoopConfiguration)
+    return number
+  }
+
+  /** Load the chunk number of src/dst vertices. */
+  def readVertexChunkNumber(): Long = {
+    val vertices_number = readVerticesNumber
+    var vertex_chunk_size = edgeInfo.getSrc_chunk_size
+    if (adjListType == AdjListType.ordered_by_dest || adjListType == AdjListType.unordered_by_dest)
```

Comment:
Please always add `{` `}`  for `if` `for` and `while` code block

Reply:
fixed